src/types/auth.ts

```diff
@@ -0,0 +1,4 @@
+export interface BasicAuth {
```

Comment:
Do we need this?

Reply:
We don't need it for this PR, I will clean it up on details page ticket.